com.visualstudio.code.insiders.yaml

```diff
@@ -0,0 +1,93 @@
+app-id: com.visualstudio.code.insiders
+default-branch: stable
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: code-insiders
+tags: [proprietary]
+desktop-file-name-suffix: ' (Flatpak)'
```

Comment:
So this is going to say `Visual Studio Code (Insiders) (Flatpak)` right? Seems a bit ungainly